src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -133,5 +136,60 @@ internal static void AddStorageAccountAttribute(Collection<CustomAttributeBuilde
             var attribute = new CustomAttributeBuilder(typeof(StorageAccountAttribute).GetConstructor(constructorTypes), constructorArguments);
             attributes.Add(attribute);
         }
+
+        protected static ICollection<JToken> ReadAsCollection(Stream valueStream)
+        {
+            byte[] bytes;
+            using (MemoryStream ms = new MemoryStream())
+            {
+                valueStream.CopyTo(ms);
+                bytes = ms.ToArray();
+            }
+
+            JArray values = null;
+            string stringValue = Encoding.UTF8.GetString(bytes);
```

Comment:
string is the lingua franca for all the script bindings. E.g. node functions produce json objects/arrays natively, and other script types like BAT, Powershell can produce JSON documents which will work the same way.